maintenance/wikia/cronjobs/sendWeeklyDigest.php

```diff
@@ -30,13 +32,27 @@ public function __construct() {
 	public function execute() {
 		$this->logRunTime();
 		$watchlistBot = new GlobalWatchlistBot();
-		$watchlistBot->sendWeeklyDigest();
+		try {
+			$watchlistBot->sendWeeklyDigest();
+		} catch ( Exception $e ) {
+			$this->logError( $e );
+		}
 	}
 
 	private function logRunTime() {
 		$message = "sendWeeklyDigest script run at " . date( "F j, Y, g:i a" ) . "\n";
 		$this->output( $message );
 	}
+
+	/**
+	 * If there's a problem when sending the weekly digest (ie, timing out), log the error. We have
+	 * a Kibana check which will send out an alert if any "Weekly Digest Error" messages are sent.
+	 */
+	private function logError( Exception $exception ) {
+		WikiaLogger::instance()->error( 'Weekly Digest Error', [
```

Comment:
Awesome, thanks @garthwebb. I also included it as part of the Weekly Digest dashboard: https://kibana.wikia-inc.com/#/dashboard/elasticsearch/Weekly%20Digest, but having our error logging in one place will make it easier to catch problems across our different products.